lib/assets.php

```diff
@@ -30,8 +30,10 @@ function asset_path($filename) {
     $manifest = [];
   }
 
-  if (array_key_exists($filename, $manifest)) {
-    return get_template_directory_uri() . '/dist/' . $manifest[$filename];
+  if (WP_ENV !== 'development' && array_key_exists($file, $manifest)) {
```

Comment:
@swalkinshaw @retlehs changed this up from what you guys had. This makes a lot more sense to me. Now instead of 3 testcases there is only 2.